src/main/webapp/i18n/de/exerciseAssessmentDashboard.json

```diff
@@ -19,20 +33,24 @@
             "gradingInstructions": "Bewertungsanweisungen",
             "problemStatement": "Aufgabenstellung",
             "exampleSolution": "Beispiellösung",
+            "exampleSolutionExplanation": "Beispiellösungserklärung",
             "close": "Schließen",
             "readAndUnderstood": "Ich habe die Anleitung gelesen und verstanden.",
+            "toRead": "zum Lesen",
+            "toReview": "zur Überprüfung",
             "totalExampleSubmissions": "Es gibt {{total}} Beispielabgaben: {{toReview}} zur Überprüfung und {{toAssess}} zur Bewertung.",
             "reviewSubmissions": "Überprüfung von Beispielabgaben",
             "start": "",
             "continue": "",
             "reviewingExampleSubmissions": "Beispielabgaben ansehen",
             "reviewExampleSubmission": "Geprüfte Beispielabgabe",
             "tutorialSubmissions": "Bewertung von Beispielabgaben",
-            "assessingExampleSubmission": "Beispielabgabe bewerten",
+            "assessingExampleSubmission": "Beispielabgaben bewerten",
             "assessExampleSubmission": "Bewertete Beispielabgabe",
             "studentsSubmissions": "Abgaben der Studierenden",
```

Comment:
"Abgaben der Studierenden" is the term that should be used for studentsSubmission in German. It would be great if  you can check the file and refactor the term "Studenten Abgaben" that you have used, I've seen it at least once ;)

Reply:
done. 👍  @lschlesinger